src/AbstractDifferentiation.jl

```diff
@@ -85,6 +85,24 @@ function jacobian(ab::HigherOrderBackend, f, xs...)
     return jacobian(lowest(ab), f, xs...)
 end
 
+"""
+    AD.second_derivative(ab::AD.AbstractBackend, f, xs...)
```

Comment:
The docstring only mentions `x` and the function only supports a single input:
```suggestion
    AD.second_derivative(ab::AD.AbstractBackend, f, x)
```